anitya/templates/project_new.html

```diff
@@ -377,7 +377,7 @@ <h1>{{ context }} project</h1>
           $(btn).show(); $(btn + "-spinner").hide();
           alert(
               'Unable to retrieve the latest version from upstream!\n'
-              + 'ERROR: ' + res.responseJSON );
+              + 'ERROR: ' + JSON.stringify(res.responseJSON) );
```

Comment:
I'm not that familiar with javascript, so this will be probably my error. Thanks for fixing this.

Reply:
You are not the only one :laughing:
This is a very common mistake in error handlers.
The default string resulting from adding an object to string is the useless `[Object object]`
and it is even easier to accidentally write that kind of code
because e.g. `console.log(object)` actually prints something useful.

Reply:
That's why people are using some JavaScript replacements on serious projects?

Reply:
It would be nice to get rid of Javascript entirely, but I'm not frontend developer, so this is not my cup of tea.